docs-kits/kits/Digital Twin Kit/page_software-operation-view.md

```diff
@@ -18,13 +18,15 @@ Digital Twins the Catena-X-way. This infrastructure empowers Data Consumers to c
 | BPN Discovery         | A microservice resolving a particular assetId against the registered BPN of its owner.                   | [Tractus-X BPN Discovery](https://github.com/eclipse-tractusx/sldt-bpn-discovery)                                                                                                                   | CX - 0053                                                   |
-| EDC Discovery         | A microservice that resolves a BPN against an EDC endpoint.                                              | [Tractus-X Portal including EDC Discovoery API](https://github.com/eclipse-tractusx/portal-backend)                                                                                                 | CX - 0001                                                   |
-| Digital Twin Registry | An exhaustive list of all Submodel Servers, with link to their assets, adhering to the AAS Registry API. | [Tractus-X Digital Twin Registry](https://github.com/eclipse-tractusx/sldt-digital-twin-registry)                                                                                                   | CX - 0002                                                   |
-| Submodel Server       | The data source adhering to a subset of the Submodel API as defined in AAS Part-2 3.0.                   | [FA³ST-Framework](https://github.com/FraunhoferIOSB/FAAAST-Service), [Eclipse Basyx](https://github.com/eclipse-basyx/basyx-java-sdk), [AASX Server](https://github.com/admin-shell-io/aasx-server) | CX - 0002                                                   |
+| EDC Discovery         | A microservice that resolves a BPN against an EDC endpoint.                                              | [Tractus-X Portal including EDC Discovery API](https://github.com/eclipse-tractusx/portal-backend)                                                                                                 | CX - 0001                                                   |
+| Digital Twin Registry | A registry for digital twins, each digital twin providing the endpoints of its submodels. <br> The registry is adhering to the AAS Registry API. | [Tractus-X Digital Twin Registry](https://github.com/eclipse-tractusx/sldt-digital-twin-registry)                                                                                                   | CX - 0002                                                   |
+| Submodel Server       | The data source adhering to a subset of the Submodel API as defined in AAS Part-2 3.0.                   | [FA³ST-Framework](https://github.com/FraunhoferIOSB/FAAAST-Service), <br> [Eclipse Basyx](https://github.com/eclipse-basyx/basyx-java-sdk), <br> [AASX Server](https://github.com/admin-shell-io/aasx-server) | CX - 0002                                                   |
```

Comment:
Line  28 and 29 make the build fail. For a line break in a table, the self-closing tag `<br/>` does that. The lone opening tag `<br>` doesn't work.